### DISCUSSION TITLE
GPU Harvester stopped working

### DISCUSSION THREAD

**Discussion by poli5** [Chia-Network/chia-blockchain#16301] [Support]
C3 compression
Version 2.0.1 and RC6
GPU p106-100 537.13 Driver
After using GPU harvesting, it started working normally, but after a few hours it stopped and couldn't earn points. I had to shut down the client and restart, but after a few hours it stopped again.
If I use the CPU, it won't be a problem, but it's too slow. My CPU is Intel 4400t.

**Comment by BrandtH22** [comment]
Hey @poli5 , as a heads up we can generally provide more thorough and timely support in our discord server (https://discord.gg/chia).
Are you seeing any errors in the log files?

Note there is an outstanding bug that leads to this experience and can be identified with a GRR error as reported here:
https://github.com/Chia-Network/chia-blockchain/issues/15404

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
